qa/build.gradle

```diff
@@ -4,7 +4,7 @@ import org.elasticsearch.gradle.test.RestIntegTestTask
 subprojects { Project subproj ->
   subproj.tasks.withType(RestIntegTestTask) {
     subproj.extensions.configure("${it.name}Cluster") { cluster ->
-      cluster.distribution = 'oss-zip'
+      cluster.distribution = System.getProperty('tests.distribution', 'oss-zip')
```

Comment:
At first I thought this PR would be **just** this line. If only things were that simple....

Reply:
And now that I've reverted most of the rest, it is almost the only line!